### PR TITLE
[SMALLFIX] Change alluxio.logger.type property to hidden

### PR DESCRIPTION
**Pull request by jiacheliu3**
This property is only used in tests. It also should not be set explicitly by the users. It's confusing to display the property on the doc page.
Screen Shot 2021-04-29 at 4 53 54 PM

### CODE REVIEW

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -333,7 +333,8 @@ public String toString() {
   public static final PropertyKey LOGGER_TYPE =
       new Builder(Name.LOGGER_TYPE)
           .setDefaultValue("Console")
-          .setDescription("The type of logger.")
+          .setDescription("This controls which logger the process uses. This is only set by code.")
```
Comment (gpang) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java:336]: Do you want to say "test code"?

Comment (jiacheliu3) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: updated

**Review by gpang — APPROVED** [review]
LGTM

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**